apps/dsb-client-gateway-api/src/app/modules/message/service/dsb-message-pooling.service.ts

```diff
@@ -194,36 +312,55 @@ export class DsbMessagePoolingService implements OnModuleInit {
         _messages.forEach((message: GetMessageResponse) => {
           if (client.readyState === WebSocket.OPEN) {
             client.send(JSON.stringify({ ...message, fqcn }));
-            this.logger.log(`[WS][sendMessagesToSubscribers][SINGLE] ${JSON.stringify({ ...message, fqcn })}`);
+            this.logger.log(
+              `[WS][sendMessagesToSubscribers][SINGLE] ${JSON.stringify({
+                ...message,
+                fqcn,
+              })}`
+            );
           }
         });
       }
-      const successAckMessageIds: AckResponse = await this.messageService.sendAckBy(_messages.map((message) => message.id).concat(idsNotAckVerify), clientId, null).catch((e) => {
-        this.logger.warn(`[WS][sendMessagesToSubscribers][sendAckBy] ${e}`);
-        return {
-          acked: [], notFound: []
-        }
-      });
-      const idsNotAck: AcksEntity[] = messages.filter(e => !successAckMessageIds.acked.includes(e.id)).map(e => {
-        return {
-          clientId: clientId,
-          messageId: e.id,
-          mbTimestamp: moment(e.timestampNanos / (1000 * 1000)).utc().toDate()
-        }
-      });
+      const successAckMessageIds: AckResponse = await this.messageService
+        .sendAckBy(
+          _messages.map((message) => message.id).concat(idsNotAckVerify),
+          clientId,
+          null
+        )
+        .catch((e) => {
+          this.logger.warn(`[WS][sendMessagesToSubscribers][sendAckBy] ${e}`);
+          return {
+            acked: [],
+            notFound: [],
+          };
+        });
+      const idsNotAck: AcksEntity[] = messages
+        .filter((e) => !successAckMessageIds.acked.includes(e.id))
+        .map((e) => {
+          return {
+            clientId: clientId,
+            messageId: e.id,
+            mbTimestamp: moment(e.timestampNanos / (1000 * 1000))
```

Comment:
Should use `luxon` DateTime instead

Reply:
Only styling changes made, I will report it to Azam